webui/react/.eslintrc.js

```diff
@@ -48,9 +55,11 @@ module.exports = {
     'indent': 'off',
     'max-len': [ 'error', 100, { tabWidth: 2 } ],
     'no-console': [ 'error', { allow: [ 'warn' ] } ],
+    'no-empty': [ 'error', { allowEmptyCatch: true } ],
     'no-multi-spaces': [ 'error', { ignoreEOLComments: true } ],
     'no-multiple-empty-lines': [ 'error', { max: 1, maxBOF: 0, maxEOF: 0 } ],
     'no-trailing-spaces': [ 'error', {} ],
+    'no-unused-vars': 'off',
```

Comment:
Why did you chose to disable this rule?

Reply:
good question, it conflicts with `@typescript-eslint/no-unused-vars` in this case based on this doc:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md#how-to-use